build/Targets/RepoToolset/Version.targets

```diff
@@ -37,21 +37,4 @@
       <AssemblyVersion Condition="'$(AssemblyVersion)' == ''">42.42.42.42</AssemblyVersion>
     </PropertyGroup>
   </Target>
-
-  <!-- 
-    Append short commit SHA to PackageVersion.
-  -->
-  <Target Name="GetPackageVersion" 
```

Comment:
@tmat Looking at this again, why did we remove this? Or was this just dead but it still worked anyways?

Reply:
https://github.com/dotnet/arcade/pull/974